libpcsxcore/gte_neon.S

```diff
@@ -290,8 +290,10 @@ FUNCTION(gteRTPT_neon): @ r0=CP2 (d,c),
 
     rtpx_preload
 
-    vmov.i32    d22, #0x7fffffff
-    vmov.i32    d23, #0x80000000
+    vmov.i32    d23, #1
+    vmov.i32    d22, #0x80000000
+    vsub.i32    d22, d22, d23
+    vmov.i32    d23, #0
```

Comment:
Is this change of behavior intended? d23 value is not the same.

Reply:
No change of behavior was intended. You can seek to remedy it as you see fit in case we snuck in an accidental mistake.